src/software/thunderscope/play/playinfo_widget.py

```diff
@@ -61,19 +63,26 @@ def refresh(self) -> None:
         for state in playinfo.play.play_state:
             play_name.append(state)
 
+        play_name.append(playinfo.play.last_guard)
+        play_name.append(playinfo.play.last_transition)
+
         for robot_id in sorted(playinfo.robot_tactic_assignment):
             robot_ids.append(robot_id)
             tactic_fsm_states.append(
                 playinfo.robot_tactic_assignment[robot_id].tactic_fsm_state
             )
-
+            tactic_guards.append(playinfo.robot_tactic_assignment[robot_id].last_guard)
+            tactic_transitions.append(playinfo.robot_tactic_assignment[robot_id].last_transition)
             tactic_names.append(playinfo.robot_tactic_assignment[robot_id].tactic_name)
+
         set_table_data(
```

Comment:
This works great! Is there a way to try to make the column lengths less jumpy? They are hard to track when using AI vs AI